File: materi/pertemuan_4/java/HitungVolume.java
import java.util.Scanner;

public class HitungVolume {
    public static void main(String[] args) {
        Scanner newScanner = new Scanner(System.in);
        System.out.println("masukan panjang: ");
        int panjang = newScanner.nextInt();

        System.out.println("masukan lebar : ");
        int lebar = newScanner.nextInt();

        System.out.println("masukan tinggi: ");
        int tinggi = newScanner.nextInt();

        double volume = panjang * lebar * tinggi;

        System.out.println(volume);
    }
}

File: materi/pertemuan_4/kotlin/hitungVolume.kt
package pertemuan_4

import java.util.*

fun main() {

    //membuat objek scanner
    val input = Scanner(System.`in`)

    //mengambil input
    print("Masukan panjang : ")
    val panjang = input.nextInt()

    print("Masukan lebar : ")
    val lebar = input.nextInt()

    print("Masukan tinggi : ")
    val tinggi = input.nextInt()

    val volume = panjang * lebar * tinggi

    //menampilkan isi variabel
    println("Hasil volume : $volume")
}